docs/conf.py

```diff
@@ -32,8 +32,8 @@
     "registry",
```

Comment:
My feeling that we can enable more modules and classes to be enabled for doc building in separate PR.